src/app.js

```diff
@@ -322,11 +327,11 @@ class Messenger extends EventEmitter {
   //////////
 
   getCacheKey(senderId/*: number */)/*: string */ {
-    return `${config.get('facebook.appId')}-${senderId}`;
+    return '' + senderId;
```

Comment:
It looks like this is hanging around solely for backwards compatability. If that's true, any reason not to add a comment to indicate we could deprecate this in a future release?